### ISSUE TITLE
There are no docs explain the meaning of learning rate parameters of DeepEmbeddingClusterMode

### ISSUE THREAD

**Issue by shendiaomo** [sql-machine-learning/models#32]
The parameters `pretrain_lr` and `train_lr` of the constructor are missing in the docstring. 
![image](https://user-images.githubusercontent.com/4180295/69804797-4d16a200-121a-11ea-98df-e851b55ae3eb.png)
